CHANGELOG.md

```diff
@@ -1,11 +1,142 @@
 # Mypy Release Notes
 
-## Unreleased
+## Next release
+
+## Mypy 1.7 [unreleased]
+
+We’ve just uploaded mypy 1.7 to the Python Package Index ([PyPI](https://pypi.org/project/mypy/)). Mypy is a static type checker for Python. This release includes new features, performance improvements and bug fixes. You can install it as follows:
+
+```
+    python3 -m pip install -U mypy
+```
+
+You can read the full documentation for this release on [Read the Docs](http://mypy.readthedocs.io).
 
 Stubgen will now include `__all__` in its output if it is in the input file (PR [16356](https://github.com/python/mypy/pull/16356)).
```

Comment:
This one could probably go in the "Other notable changes" section too

Reply:
This is actually not included in the release, so I moved it to the section about the next release (on master).